Updates for PM review. (#35)
mulesoft/docs-partner-manager: modules/ROOT/pages/configure-host.adoc

> Your organization is considered the host partner in Anypoint Partner Manager.

Reviewer: ditto standardizing how this doc refers to host and partner throughout

> Within the Host page you can do the following:

Reviewer: ```suggestion
Within the Host page, you can do the following:
```

* Add identifiers to uniquely identify yourself with your partners

* Add certificates owned by your organization

* Configure global X12 validation settings

* Add storage for your B2B message payloads

> == Configure the host

Reviewer: ```suggestion
== Configure the Host
```

Reviewer: Suggest adding a text transition between the subtitle and the first step.

> . Select the name of your organization from the navigation pane on the left-hand side.

Reviewer: left-hand > left

> From here you configure the following based on your indentifier and certificate type. This includes any global settings you need to modify, and the location of your payload storage configuration.

Reviewer: ```suggestion
. Configure the following based on your identifier and certificate type, including any global settings you need to modify and the location of your payload storage configuration:
```

// Refer to X12 Settings to change any of the default settings for all X12 received messages.

> * Add your identifiers such as the AS2 Identifier, X12 ISA, X12 GS, and/or DUNS.

Reviewer: AS2 ID

and/or > [choose just one]

* Upload any digital certificates (such as an X.509 certificate in PKCS12 file format for the AS2 transport protocol).

* Change any of the default global X12 validation, acknowledgement, control number, and character set settings.

> * Storage for your B2B message payloads. Configure the location where you store payloads containing the messages you send to and receive from your partners.

Reviewer: This bullet list item is not parallel with its peers, which are all verb-based. Reword for parallelism.

> When you are ready, you can Configure your trading partner(s).

Reviewer: Not "when you are ready"; rather, after you configure yourself as host, you can configure your trading partners.
